WORKSPACE

```diff
@@ -331,6 +331,30 @@ go_repository(
     importpath = "google.golang.org/genproto",
 )
 
+go_repository(
+    name = "com_github_hashicorp_go_rootcerts",
+    commit = "6bb64b370b90e7ef1fa532be9e591a81c3493e00",
```

Comment:
If you don't mind, can you put dates on these commits? We're not enforcing this practice, but strongly recommend to keep track of how old the dependencies are.